modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/VirtualFlowSubClassTest.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2010, 2016, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Minor nit: Since this is a new file, it should be just `Copyright (c) 2018, Oracle ...`